src/addrman.h

```diff
@@ -176,6 +177,20 @@ friend class CAddrManTest;
     mutable RecursiveMutex cs;
 
 private:
+    //! Serialization versions.
+    enum Format : uint8_t {
+        V0_HISTORICAL = 0,    //!< historic format, before commit e6b343d88
+        V1_DETERMINISTIC = 1, //!< for pre-asmap files
+        V2_ASMAP = 2,         //!< for files including asmap version
+        V3_BIP155 = 3,        //!< same as V2_ASMAP plus addresses are in BIP155 format
+    };
+
+    //! The initial value of a field that is incremented every time an incompatible format
+    //! change is made. This is 32 instead of 0 because we overtook the "key size" field
+    //! which was 32 historically and we don't want to start with 0 and bump into 32 at
+    //! some point in the future.
```

Comment:
"we don't want to bump into 32" isn't the reason we start at 32. The reason is that v0.20 and earlier wrote 32 into this field, so if we didn't use this offset, then minimum_version would be greater than the current version and v0.21 software wouldn't be able to read old peers.dat files. I'd just leave the explanation out.

Reply:
> I'd just leave the explanation out.

Removed the last part of the comment.